docs/source/tools/visual.rst

```diff
@@ -0,0 +1,32 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+DAML Contract Visulization
```

Comment:
Task-based headings are normally better for users - so something like "Visualizing DAML contracts"

(Plus you don't need to use Title Case, just capitalise like you would in a sentence)

Reply:
There's also a typo in "Visulization".

Reply:
:(